documentation/schema.json

```diff
@@ -1649,6 +1660,36 @@
         }
       }
     },
+    "DotnetMonitorDebugOptions": {
+      "type": "object",
+      "description": "Configuration options for debugging dotnet-monitor features.\nThese options are not officially supported and are subject to change.",
+      "additionalProperties": false,
+      "properties": {
+        "Exceptions": {
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/ExceptionsDebugOptions"
+            }
+          ]
+        }
+      }
+    },
+    "ExceptionsDebugOptions": {
```

Comment:
Note that any properties under `DotnetMonitorDebug` are **not** documented as they are not officially supported and are subject to change.